### ISSUE TITLE
Batch norm and image normalization in struct2depth

### ISSUE THREAD

**Issue by Yevkuzn** [tensorflow/models#7604]
- **What is the top-level directory of the model you are using**: models/research/struct2depth
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: of course
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Fedora 30
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.11
- **Bazel version (if compiling from source)**: -
- **CUDA/cuDNN version**: 9.2 / 7.2.1
- **GPU model and memory**: 1080 TI
- **Exact command to reproduce**: python inference.py ...

Hi there,
I noticed that there is a connection between image normalization and the way batch norm is used during inference.

The issue #7343 indeed describes a bug and the image is not normalized before being fed to a neural network. This causes the effect described in issue #6927. 

If using batch norm in train mode, the features are normalized using current batch statistics, which allows to compensate for absent image normalization. On the other hand, in test mode batch normalization is done with fixed statistics, which causes degraded performance on not normalized images.

If image normalization is fixed (e.g. as in #7342) and batch norm mode is set to test, quantitative results improve compared to those without image normalization and train mode batch norm.

BR
Yevhen
